Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:22.04 as build
+
+LABEL org.opencontainers.image.authors="Santiago <san@san.com>"
+LABEL org.opencontainers.image.source="https://github.com/santaclose/ssbd"
+
+# hadolint ignore=DL3008
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+        binutils-mips-linux-gnu \
+        build-essential \
+        clang \
+        # FIXME: overkill
+        gcc-multilib \
+        git \
+        libcapstone-dev \
+        # NOTE: used by viewText.sh
+        nano \
```

Comment:
we don't need nano or xxd to build (unsure about build-essential and others)

Reply:
Yea, I did it this way to also prepare for a development container, and wanted to define all the tools actually used in the codebase.